win-package.py

```diff
@@ -0,0 +1,90 @@
+# This file should not be used directly but using build.bat
+
+import os
+import os.path
+from os.path import join, normpath, basename
+import platform
+import re
+import tempfile
+import shutil
+import glob
+import tarfile
+from distutils import dir_util
+
+from precise import __version__
+
+
+import hashlib
+def filemd5(fname):
+    hash_md5 = hashlib.md5()
+    with open(fname, "rb") as f:
+        for chunk in iter(lambda: f.read(4096), b""):
+            hash_md5.update(chunk)
+    return hash_md5.hexdigest()
+
+def make_tarfile(output_filename, source_dir):
+    with tarfile.open(output_filename, "w:gz") as tar:
+        tar.add(source_dir, arcname=basename(source_dir))
+
+def tar_name(tar_prefix):
+    arch = platform.machine()
+    return "{tar_prefix}_{version}_win_{archname}.tar.gz".format(
+                                                            tar_prefix=tar_prefix,
+                                                            version=__version__,
+                                                            archname = arch)
+
+def filecontains(path, string):
+    try:
+        f = open(path)
+        content = f.read()
+        f.close()
+        return string in content
+    except FileNotFoundError:
+        return False
+
+def package_scripts(tar_prefix, combined_folder, scripts, train_libs):

```

Comment:
it looks like it could be useful to split this function up a little bit for readability

Reply:
Ok, can do.
Basically it's a port of build.sh, but why not :)